suave/sol/standard_peekers/example.sol

```diff
@@ -44,6 +44,44 @@ contract ExampleEthCallSource {
 
         return abi.encodeWithSelector(this.emptyCallback.selector);
     }
+
+    event LogCallback(bytes data);
+
+    function emitLogCallback(uint256 num) public {
+        // From the msg.input, the 'confidential context' sequence
+        // starts at index 37 (4 signbature bytes + 32 bytes for uint256 + 3 bytes for the magic sequence)
+        uint256 magicSequenceIndex = 4 + 32 + 3;
```

Comment:
Would you also want to make sure that msg.data[magicSequenceIndex] is equal to the magic sequence bytes?

Reply:
This is only for an e2e test to make sure that the encoded appended date (we have to find a new name for this) can be decoded later. `suave-std` which is the production side of things for the Solidity counterpart does more checks. https://github.com/flashbots/suave-std/pull/56/files#diff-c3373d1c6d1df488550173a8c39686af3c68cf6c525b7ca2446afa8dbcef95fbR36